dns/README.md

```diff
@@ -58,18 +58,21 @@ future changes by simply running `terraform apply
 ## Inputs ##
 
 | Name | Description | Type | Default | Required |
-|------|-------------|:----:|:-------:|:--------:|
-| aws_region | The AWS region where the non-global resources for the DNS account are to be created (e.g. us-east-1). | string | `us-east-1` | no |
-| provisionaccount_role_description | The description to associate with the IAM role that allows sufficient access to provision all AWS resources in the DNS account. | string | `Allows sufficient access to provision all AWS resources in the DNS account.` | no |
-| provisionaccount_role_name | The name to assign the IAM role that allows sufficient permissions to provision all AWS resources in the DNS account. | string | `ProvisionAccount` | no |
-| tags | Tags to apply to all AWS resources created. | map(string) | `{}` | no |
-| users_account_id | The ID of the users account.  This account will be allowed to assume the role that allows sufficient access to provision all AWS resources in the DNS account. | string | | yes |
+|------|-------------|------|---------|:-----:|
+| aws_region | The AWS region where the non-global resources for the DNS account are to be provisioned (e.g. "us-east-1"). | `string` | `"us-east-1"` | no |
+| provisionaccount_role_description | The description to associate with the IAM role that allows sufficient permissions to provision all AWS resources in the DNS account. | `string` | `"Allows sufficient permissions to provision all AWS resources in the DNS account."` | no |
```

Comment:
I don't think you need quotes AND backticks for these default values.

Reply:
This is the output from `terraform-docs markdown --no-escape .` which I assumed we were using.  Cause I was sure these were not artisanal cut and paste jobbies.  ;)

Reply:
Taking a look at using that pre-comit hook @dav3r was experimenting with: https://github.com/antonbabenko/pre-commit-terraform#notes-about-terraform_docs-hooks

Reply:
Modified in 32ce711

I've removed the double quoting that `terraform-doc` was outputting.  This is now replaced with the output of:
```console
terraform-docs markdown --no-escape --no-header . | sed -E 's/`"|"`/`/g'
```